finance_dl/gemini.py

```diff
@@ -0,0 +1,241 @@
+"""Retrieves trades, transfers, balances and price information from the Gemini Crypto Exchange.
+
+Configuration:
+==============
+
+The following keys may be specified as part of the configuration dict:
+
+- `credentials`: Required.  Must be a `dict` with `'api_key'`, `'api_secret'` keys.
+
+- `output_directory`: Required.  Must be a `str` that specifies the path on the
+  local filesystem where the output will be written.  If the directory does not
+  exist, it will be created.
+
+Output format:
+==============
+
+CSV files.
+
+output_directory
+-- trades.csv
+-- transfers.csv
+-- balances.2021-06-01.csv
+-- balances.2021-07-01.csv
+...
+
+
+Example:
+========
+
+    def CONFIG_gemini():
+        return dict(
+            module='finance_dl.gemini',
+            credentials={
+                'key': 'XXXXXX',
+                'secret': 'XXXXXX',
+            },
+            output_directory=os.path.join(data_dir, 'gemini'),
+        )
+
+
+Interactive shell:
+==================
+
+From the interactive shell, type: `self.run()` to start the scraper.
```

Comment:
Your PR description says something about interactive mode not being supported, that seems inconsistent with this? I'm not sure what that means, though, as far as I know scrapers don't need to do anything special to support interactive mode. Maybe I'm misunderstanding what you meant by not supporting interactive mode.

Reply:
They do require more work. You need to pop a Python interpreter for the user. I'll just mark it as not implemented and leave it for another day.